pkgs/development/compilers/go/bootstrap118-gentoo.nix

```diff
@@ -0,0 +1,11 @@
+# Only used for RISC-V
+{ callPackage }:
+callPackage ./binary.nix {
+  version = "1.18.6";
+  hashes = {
+    linux-riscv64 = "sha256-I3M142J1ifK1rUQWe4kgwwMhaKoa2MLc92RjuATGD2c=";
+  };
+  urls = {
+    linux-riscv64 = "https://dev.gentoo.org/~williamh/dist/go-linux-riscv64-bootstrap-1.18.6.tbz";
```

Comment:
Long term it would be nice if this would become a package that one can rebuild themself from source and is uploaded to tarballs.nixos.org.

Reply:
Go upstream is going to start bumping bootstrap versions frequently and even although go is quick to build I don't really want to get into needing a bootstrap chain from a version that we uploaded.